styles/_base.scss

```diff
@@ -117,48 +117,107 @@ ul, ol {
 
   #contentFrame {
     position: relative;
-    padding-top: $barLg;
     overflow-y: scroll;
     height: calc(100% - #{$bar});
     box-sizing: border-box;
     z-index: 0;
 
-    #pageTabBar {
-      position: fixed;
-      top: $bar;
-      left: 0;
-      z-index: 1;
-      width: 100%;
-      height: $barLg;
+    &:before {
+      // bottom border for the chat close button, it gets covered by the pageTabBar
+      content: '';
+      display: block;
+      width: $chatCloseBtn;
+      height: 0;
       border-bottom-width: 1px;
       border-bottom-style: solid;
+      position: absolute;
+      top: $chatCloseBtnTop - $bar;
+      right: -$chatCloseBtn;
+      width: $chatCloseBtn;
+      height: $chatCloseBtn;
+      @include chatOpeningTransition(right);
+      z-index: 0;
+    }
+
+    .chatOpen &:before {
+      right: 0;
+    }
 
-      .pageTabs {
-        max-width: $pageWidth;
-        margin: 0 auto;
+    #pageContainer {
+      padding-top: $barLg;
+
+      #pageTabBar {
+        position: fixed;
+        top: $bar;
+        left: 0;
+        z-index: 1;
+        width: 100%;
+        height: $barLg;
+        overflow: hidden;
+
+        &:before {
+          // mask for the chat close button
+          content: '';
+          display: block;
+          width: $chatCloseBtn;
+          height: $chatCloseBtn;
+          position: absolute;
+          top: $chatCloseBtnTop - $bar;
+          right: -$chatCloseBtn;
+          @include chatOpeningTransition(right);
+          z-index: 0;
+        }
 
-        .userIcon {
-          border-width: 2px;
-          border-style: solid;
-          width: 44px;
-          height: 44px;
-          margin-right: $pad;
+        &:after {
+          // false bottom border for the pageTab
```

Comment:
To allow the page tab border to move when the chat is open, so it doesn't go through the transparent close button, it's been replaced with a 1px high div.